Header header
float64 u
float64 v
float64 w
float64 temp
float64 mag
